src/pages/caseDetails/caseDetailsView.js

```diff
@@ -436,57 +456,61 @@ const CaseDetail = ({ classes, data }) => {
           ? (
             <Grid container spacing={4} className={classes.tableDiv}>
               <Grid item xs={12}>
-                <GridWithFooter
-                  data={data.samplesByCaseId}
-                  title={(
-                    <div className={classes.tableTitle}>
-                      <span className={classes.tableHeader}>{table1.tableTitle}</span>
-                    </div>
-                      )}
-                  columns={getColumns(table1, classes, data, externalLinkIcon)}
-                  options={getOptions(table1, classes)}
-                  customOnRowsSelect={table1.customOnRowsSelect}
-                  openSnack={openSnack}
-                  closeSnack={closeSnack}
-                  disableRowSelection={table1.disableRowSelection}
-                  buttonText={table1.buttonText}
-                  saveButtonDefaultStyle={table1.saveButtonDefaultStyle}
-                  ActiveSaveButtonDefaultStyle={table1.ActiveSaveButtonDefaultStyle}
-                  DeactiveSaveButtonDefaultStyle={table1.DeactiveSaveButtonDefaultStyle}
-                  tooltipMessage={table1.tooltipMessage}
-                  tooltipContent={tooltipContent}
-                  showtooltip
-                />
-              </Grid>
-            </Grid>
-          ) : ''}
-
-        {table2.display
-          ? (
-            <div id="table_case_detail_samples" className={classes.tableContainer}>
-              <Grid container spacing={4} className={classes.tableDiv}>
-                <Grid item xs={12}>
+                <MuiThemeProvider theme={computedTheme}>
                   <GridWithFooter
-                    data={files}
+                    data={data.samplesByCaseId}
                     title={(
                       <div className={classes.tableTitle}>
-                        <span className={classes.tableHeader}>{table2.tableTitle}</span>
+                        <span className={classes.tableHeader}>{table1.tableTitle}</span>
                       </div>
                       )}
-                    columns={getColumns(table2, classes, data)}
-                    options={getOptions(table2, classes)}
-                    customOnRowsSelect={table2.customOnRowsSelect}
+                    columns={getColumns(table1, classes, data, externalLinkIcon)}
```

Comment:
Do we have any changes in bento-tools?

Reply:
Only the style change, no thing bento-tool related.